assets/src/dashboard/app/index.js

```diff
@@ -38,6 +39,39 @@ import {
   SavedTemplatesView,
 } from './views';
 
+const AppContent = () => {
+  const {
+    state: { currentPath },
+  } = useContext(RouterContext);
```

Comment:
Pull out this logic to a mini-helper component just so I could access the router context.